src/style.css

```
@@ -0,0 +1,106 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+  color: #3d4e61;
+  font-weight: 300;
+}
```

color change in selector
```suggestion
@font-face {
  font-family: "Avenir";
  src: url("./fonts/avenir-book.ttf");
  font-weight: 300;
}

@font-face {
  font-family: "Avenir";
  src: url("./fonts/avenir-heavy.ttf");
  font-weight: 800;
}
```